### PR TITLE
NPF: two small design fixes

### PR DESCRIPTION
**Pull request by regiskuckaertz**
## Why are you doing this?

Font and text updated as per documented in the Trello card.

[**Trello Card**](https://trello.com/c/U8ggGUNJ/885-npf-landing-page-interaction-design-copy-tweaks-2-2)

## Screenshots

screen shot 2018-09-24 at 15 20 38

screen shot 2018-09-24 at 16 12 06

### CODE REVIEW

**Comment by regiskuckaertz** [comment]
@ionamckendrick @Amohkhan should there be a non-breakable space in "The Guardian"?

File: assets/pages/new-contributions-landing/contributionsLanding.scss [hunk 1]
```diff
@@ -183,8 +183,7 @@ button {
 }
 
 .blurb {
-  font-size: 14px;
-  line-height: 1.4;
+  font: 100 16px/1.25 $gu-headline;
```
Comment (joelochlann) [inline assets/pages/new-contributions-landing/contributionsLanding.scss]: what is the unitless `100` font-size?

File: assets/pages/new-contributions-landing/components/ContributionThanks.jsx [hunk 2]
```diff
@@ -29,7 +29,7 @@ const mapStateToProps = state => ({
 function ContributionThanks(props: PropTypes) {
   return (
     <div className="gu-content__content">
-      <h1>Thank you for a valuable contribution</h1>
+      <h1 class="header">Thank you for a valuable contribution</h1>
```
Comment (joelochlann) [inline assets/pages/new-contributions-landing/components/ContributionThanks.jsx]: 👍

File: assets/pages/new-contributions-landing/contributionsLanding.scss (hunk 1 above)
Comment (regiskuckaertz) [reply assets/pages/new-contributions-landing/contributionsLanding.scss]: It's the font-weight, sadly browsers don't agree on what the keywords like `lighter` mean exactly. And with variable fonts, it's going to be https://medium.com/@clagnut/get-started-with-variable-fonts-c055fd73ecd7 🤣

**Pull request merged**